packages/kotti-ui/source/make-props.ts

```diff
@@ -113,10 +122,14 @@ const walkSchemaTypes = <SCHEMA extends z.ZodTypeAny>(
 		case z.ZodFirstPartyTypeKind.ZodBoolean:
 		case z.ZodFirstPartyTypeKind.ZodDate:
 		case z.ZodFirstPartyTypeKind.ZodFunction:
+		case z.ZodFirstPartyTypeKind.ZodLiteral:
```

Comment:
try removing it - might be unneeded if all direct Literal types were now changed to NativeEnum